packages/terra-core-docs/src/terra-dev-site/doc/toggle-button/example/ButtonAttrsToggleButton.jsx

```diff
@@ -2,15 +2,42 @@ import React from 'react';
 import Button from 'terra-button';
 import ToggleButton from 'terra-toggle-button';
 
-const ButtonAttrsToggleButton = () => (
-  <ToggleButton
-    closedButtonText="Button Emphasis ToggleButton"
-    buttonAttrs={{
-      variant: Button.Opts.Variants.EMPHASIS,
-    }}
-  >
-    <p>CommonWell Health Alliance, integrated with the Cerner clinical EHR platform, facilitates nationwide interoperability. CommonWell provides infrastructure to help identify patients as they transition through care facilities; locate health records regardless of where care occurred; and retrieve relevant care documents in an efficient manner.</p>
-  </ToggleButton>
-);
+const ButtonAttrsToggleButton = () => {
+  const handleLearnMoreClick = () => {
+    alert('Learn More Button');
+  };
+
+  const handleContactClick = () => {
+    alert('Contact Button Click');
+  };
+
+  return (
+    <ToggleButton
+      closedButtonText="Learn More"
+      buttonAttrs={{
+        variant: Button.Opts.Variants.EMPHASIS,
+      }}
+      onClose={handleLearnMoreClick}
+    >
+      <div>
+        <h2>Integrated Healthcare Solutions</h2>
+        <p>
+          Our healthcare system offers state-of-the-art integrated solutions that enhance patient care and improve outcomes. With our advanced electronic health records (EHR) platform and interoperability infrastructure, we aim to revolutionize the way healthcare information is shared and utilized.
+        </p>
+        <p>
+          By seamlessly connecting healthcare providers, laboratories, pharmacies, and patients, we ensure that critical health information is readily available at the point of care. Our integrated solutions streamline workflows, reduce errors, and enable informed decision-making for healthcare professionals.
+        </p>
+        <p>
+          Our platform supports nationwide interoperability, allowing healthcare facilities to access comprehensive patient records regardless of where care was provided. Through our partnership with the CommonWell Health Alliance, we facilitate secure data exchange and provide a unified view of patient information.
+        </p>
+        <p>
+          To learn more about our integrated healthcare solutions and how they can benefit your organization, please
+          <Button text="Contact us" onClick={handleContactClick} />
```

Comment:
was these button added on purpose ..?? Instead we can just include anchor pointing to '#'
```suggestion
     <a href="#"> Contact us </a>
```

Reply:
Added fix on 'https://github.com/cerner/terra-core/pull/3827/commits/a151c1bdafa8ea0129148661770845c8e3fcbfba'